updateClient.js

```diff
@@ -0,0 +1,43 @@
+const updateClient = function(client, id, connection) {
+  return new Promise(function(resolve, reject){
+    const query = 'UPDATE Clients SET ? WHERE id = ?';
+    connection.query(', [client, id], function (error, results, fields) {
+      if (!error) {
+        resolve(results);
```

Comment:
The promise is missing a reject in case of error. Nodejs way is to deal with errors first and then handle success scenario